test/rulesets/Base/base.jl

```diff
@@ -100,8 +106,8 @@
     end
 
     @testset "mod" begin
-        test_frule(mod, 10rand(), rand())
-        test_rrule(mod, 10rand(), rand())
+        test_frule(mod, rand_mod_pair()...)
+        test_rrule(mod, rand_mod_pair()...)
```

Comment:
I would rather hard code this with literals.
Tests are clearer when we use literals.
A lot of are scalar tests have already been rewritten to use literals.

and in this case the mode code adds much complexity for little gain.

Reply:
Ok, I can change that